public/containers/RefreshAction/index.tsx

```diff
@@ -0,0 +1,172 @@
+
+  useEffect(() => {
+    if (!!services && visible) {
+      switch (type) {
+        case INDEX_OP_TARGET_TYPE.INDEX:
+          filterBlockedItems<CatIndex>(services, selectedItems as CatIndex[], IndexOpBlocksType.Closed, indexBlockedPredicate)
+            .then((filteredStreamsResult) => {
+              setUnBlockedItems(filteredStreamsResult.unBlockedItems.map((item) => item.index));
+              setBlockedItems(filteredStreamsResult.blockedItems.map((item) => item.index));
+            })
+            .catch(() => {
+              /*
+            It's not a critical error although if it fails unlikely other call will succeed,
+            set unblocked items to all so we won't filter any of the selected items.
+             */
+              setUnBlockedItems(selectedItems.map((item: CatIndex) => item.index));
+            });
+          break;
+        case INDEX_OP_TARGET_TYPE.ALIAS:
+          filterBlockedItems<IAlias>(services, selectedItems as IAlias[], IndexOpBlocksType.Closed, aliasBlockedPredicate)
+            .then((filteredDataStreamsResult) => {
+              setUnBlockedItems(filteredDataStreamsResult.unBlockedItems.map((item) => item.alias));
+              setBlockedItems(filteredDataStreamsResult.blockedItems.map((item) => item.alias));
+            })
+            .catch(() => {
+              /*
+            It's not a critical error although if it fails unlikely other call will succeed,
+            set unblocked items to all so we won't filter any of the selected items.
+             */
+              setUnBlockedItems(selectedItems.map((item: IAlias) => item.alias));
+            });
+          break;
+        case INDEX_OP_TARGET_TYPE.DATA_STREAM:
+          filterBlockedItems<DataStream>(services, selectedItems as DataStream[], IndexOpBlocksType.Closed, dataStreamBlockedPredicate)
+            .then((filteredDataStreamsResult) => {
+              setUnBlockedItems(filteredDataStreamsResult.unBlockedItems.map((item) => item.name));
+              setBlockedItems(filteredDataStreamsResult.blockedItems.map((item) => item.name));
+            })
+            .catch(() => {
+              /*
+            It's not a critical error although if it fails unlikely other call will succeed,
+            set unblocked items to all so we won't filter any of the selected items.
+             */
+              setUnBlockedItems(selectedItems.map((item: DataStream) => item.name));
+            });
+          break;
+      }
+    } else {
+      setUnBlockedItems([]);
+      setBlockedItems([]);
+    }
+  }, [visible, services, type, selectedItems]);
+
+  const onConfirm = useCallback(async () => {
+    if (!!services) {
+      const result = await services.commonService.apiCaller({
+        endpoint: "indices.refresh",
+        data: {
+          index: unBlockedItems.join(","),
+        },
+      });
+      if (result && result.ok) {
+        let message = `Refresh ${type} [${unBlockedItems.join(",")}] successfully`;
```

Comment:
```suggestion
        let message = `Refresh ${type} [${unBlockedItems.join(", ")}] successfully`;
```

It would be better that we add a space to make the message truncatable.

Reply:
Updated